history.go

```diff
@@ -51,6 +51,33 @@ func (h *History) Newer(buf *Buffer) (new *Buffer, changed bool) {
 	return new, true
 }
 
+// Get x lines back in the history as a string array
```

Comment:
Godoc comments should start function name like `GetLines returns ....` .
https://blog.golang.org/godoc-documenting-go-code